coreos-assembler

```diff
@@ -12,6 +12,16 @@ case $USER in
     *) echo "Executed as non-builder user; assuming sudo rights..." 1>&2;;
 esac
 
+# Ensure we've unshared our mount namespace so
+# the later umount doesn't affect the host potentially
+if [ -z "${coreos_assembler_unshared:-}" ]; then
```

Comment:
is the `:-` really needed ?

Reply:
For the the `u` in `set -euo pipefail` at the top, yes.